dist/panzoom.js

```diff
@@ -175,19 +175,16 @@ function createPanZoom(domElement, options) {
     var boundingBox = getBoundingBox()
     if (!boundingBox) return
 
-    var adjusted = false
```

Comment:
Could you please elaborate why was this removed?

Reply:
Hi anvaka
It is related to the change in these [two lines](https://github.com/anvaka/panzoom/pull/15/commits/5bcad6ef7d2f9f2ba6804205ebfcbbb92469bc84#diff-168726dbe96b3ce427e7fedce31bb0bcL286). the was disabled in case the transform was adjusted in the keepTransformInsideBounds() and this results in weird behavior in panzoom when the bounds set to true and the boundsPadding set to a value new or equal 1. I tried it with dom element and when the boundsPadding is 1, no zoom will be applied.

Removing this condition has no negative effect on the general behaviour of panzoom (I tested it on the dom element) and it allowed bounds to work properly and since the returned value is not needed when this condition is removed, I deleted the returned value in the function keepTransformInsideBounds()